autoload.php.dist

```diff
@@ -51,31 +64,45 @@ class DeprecationErrorHandler
             }
         } else {
             self::$isRegistered = true;
-            register_shutdown_function(function () use (&$deprecations) {
-                if ($deprecations[0]) {
-                    if (function_exists('posix_isatty') && @posix_isatty(STDOUT)) {
-                        echo "\n\x1B[43;30mDeprecation notices ($deprecations[0]):\x1B[0m\n";
-                    } else {
-                        echo "\nDeprecation notices ($deprecations[0]):\n";
-                    }
+            register_shutdown_function(function () use (&$deprecations, $deprecationHandler) {
+
+                if (function_exists('posix_isatty') && @posix_isatty(STDOUT)) {
```

Comment:
what about Windows users with ConEmu (I care less than previously though as I'm not running tests on Windows anymore and this is only about the testsuite, but it would be handy) ?

Btw, isn't it possible to get this info from PHPUnit itself (using colors if PHPUnit is using colors) ?